CONTRIBUTING.md

```diff
@@ -15,6 +15,21 @@ and [submitting pull requests](#pull-requests).
 <a name="issue-labels"></a>
 ## Issue labels
 
+* `bug` - Report any bugs regarding postcss.com, please be as specific as possible.
+* `discussion` - Open discussion, please chime in! (E.g. for upcoming features, questions, planning, etc.)
+* `feature request` - Request for a feature or enhancement.
+* `help wanted` - Issues we need help to resolve.
+* `high-priority` - High priority issues that need to be resolved as soon as possible.

```

Comment:
Actually, maybe lets remove the `High priority` and `Low priority` from the descriptions totally and change them to "Issues that need to be resolved asap" and "Issues that don't need urgent solving" or something like that?